config/routes.rb

```diff
@@ -427,13 +420,9 @@
     get '/bulk' => 'bulk#destroy'
   end
 
-  resources :work_packages, only: [:update, :index] do
-    get :new_type, on: :member
-
+  resources :work_packages, only: [:index] do
     get :column_data, on: :collection # TODO move to API
 
-    resources :relations, controller: 'work_package_relations', only: [:create, :destroy]
```

Comment:
Done in updated https://github.com/oliverguenther/openproject/commit/764d54dc698ae0649a5354d8103386dbe6fd67ff